Tests/L0/AzureRmWebAppDeployment/AzureRMUtil.js

```diff
@@ -0,0 +1,42 @@
+function getAzureRMWebAppPublishProfile(SPN, webAppName, resourceGroupName, deployToSlotFlag, slotName) {
+	
+	var mockPublishProfile = {
+		profileName: 'mytestapp - Web Deploy',
+ 		publishMethod: 'MSDeploy',
+		publishUrl: 'mytestappKuduUrl',
+		msdeploySite: 'mytestapp',
+		userName: '$mytestapp',
+		userPWD: 'mytestappPwd',
+		destinationAppUrl: 'mytestappUrl',
+		SQLServerDBConnectionString: '',
+		mySQLDBConnectionString: '',
+		hostingProviderForumLink: '',
+		controlPanelLink: '',
+		webSystem: 'WebSites' 
+	};
+
+	if(deployToSlotFlag) {
+		
+		mockPublishProfile.profileName =  'mytestapp-' + slotName + ' - Web Deploy';
+		mockPublishProfile.publishUrl = 'mytestappKuduUrl-' + slotName;
+		mockPublishProfile.msdeploySite = 'mytestapp__' + slotName;
+		mockPublishProfile.userName = '$mytestapp__' + slotName;
+		mockPublishProfile.userPWD = 'mytestappPwd';
+		mockPublishProfile.destinationAppUrl = 'mytestappUrl-' + slotName;
+		
+	}
+
+	return mockPublishProfile;
+
+}
+exports.getAzureRMWebAppPublishProfile = getAzureRMWebAppPublishProfile;
```

Comment:
Don't use this syntax for exporting use export function ABC(): <returnType> { 

}